cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -59,6 +59,19 @@ These ciphers can't be carried over from CloudHub to CloudHub 2.0:
 Limitation for Restricting Outbound Egress Traffic::
 CloudHub VPC doesn't restrict outbound egress traffic, so removing the default firewall rule that allows this traffic can cause failures in your migrated private space.
 
+
+== Understand Environment Association
+
+During the upgrade process from CloudHub VPC to CloudHub 2.0 Private Space, the environments associated with your VPC stays in sync with the environments associated with the upgraded CloudHub 2.0 private space to avoid configuration drift.
```

Comment:
```suggestion
During the upgrade process from CloudHub VPC to CloudHub 2.0 Private Space, the environments associated with your VPC stay in sync with the environments associated with the upgraded CloudHub 2.0 Private Space to avoid configuration drift.
```